manifests/v2.1.0/vsphere-7.0u1/deploy/create-validation-webhook.sh

```diff
@@ -49,7 +49,7 @@ kubectl delete validatingwebhookconfiguration.admissionregistration.k8s.io valid
 kubectl delete serviceaccount vsphere-csi-webhook --namespace "${namespace}" 2>/dev/null || true
 kubectl delete clusterrole.rbac.authorization.k8s.io vsphere-csi-webhook-role 2>/dev/null || true
 kubectl delete clusterrolebinding.rbac.authorization.k8s.io vsphere-csi-webhook-role-binding --namespace "${namespace}" 2>/dev/null || true
-kubectl delete deployment vsphere-csi-webhook --namespace "${namespace}" || true
+kubectl delete deployment vsphere-csi-webhook --namespace "${namespace}" 2>/dev/null || true
```

Comment:
@cormachogan can we also update this file - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/manifests/dev/vsphere-7.0u2/vanilla/create-validation-webhook.sh

Reply:
Done!